### PR TITLE
Secrets Management

### PR DESCRIPTION
**Pull request by Tayef-Shah**
Just a quick note: There's a utility which allows you to automatically update your secrets with every git pull, but it doesn't work when you rebase. If we are rebasing like on SVP, the additional utility won't be of much use.

### CODE REVIEW

**Comment by danielchen115** [comment]
Not sure if it's just github's code viewer, but it looks like the indents are more than the usual 4 spaces 😮 Running the linter should fix it

**Comment by Tayef-Shah** [comment]
In my code editor I'm only seeing 4 spaces. I ran the linter anyway.

File: README.md [hunk 1]
```diff
@@ -6,8 +6,13 @@
 
 1. Clone the repo
 2. Run `docker-compose up -d` in the project directory
-3. Copy `.env.example` as `.env` in `projectread/` and add the environment variables
-4. Go to `localhost:8000` to make sure everything works
+3. Copy `.env.example` as `.env` in `projectread/`
+4. Set up your local vault client by following the sections *Install the Vault Client* and *Point to our Vault server* in these [docs](https://www.notion.so/uwblueprintexecs/Secret-Management-2d5b59ef0987415e93ec951ce05bf03e#86337406f266493d990911901480f435)
```
Comment (brittanylau) [inline README.md]: I also had to do the "authenticate" step to log in using my PAT! would you mind adding that?

File: secret.config [hunk 2]
```diff
@@ -0,0 +1 @@
+ENV_KEYS=projectread/.env
```
Comment (brittanylau) [inline secret.config]: could we add an empty line at the end of this file?

**Review by brittanylau — APPROVED** [review]
successfully followed the instructions to pull down secrets, and also add new ones! thanks for setting this up for our team :)

super cool stuff @uwblueprint/internal-tools! 🌟

File: update_secret_files.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import sys
```
Comment (brittanylau) [inline update_secret_files.py]: could we put `setup.sh` and this file in a directory named `scripts`?

**Pull request merged**